Blockzilla/PhotonActionSheet.swift

```diff
@@ -0,0 +1,596 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
```

Comment:
If this code is ported over from Firefox iOS (it appears to be) could we throw a comment mentioning that at the top of this?